sequencing-server/src/lib/codegen/CommandEDSLPreface.spec.ts

```diff
@@ -269,5 +270,30 @@ describe('Sequence', () => {
     ],
   });`);
     });
+
+    it('should convert with Hardware,', () => {
+      const sequence = Sequence.new({
+        seqId: 'HW',
+        metadata: {},
+        hardware_commands: [
+          HardwareStem.new({ stem: 'HW_PYRO_DUMP' }).DESCRIPTION('Fire the pyros').METADATA({ author: 'Emery' }),
+        ],
+      });
+
+      expect(sequence.toEDSLString()).toEqual(
+        'export default () =>\n' +
+          '  Sequence.new({\n' +
+          "    seqId: 'HW',\n" +
+          '    metadata: {},\n' +
+          '    hardware_commands: [\n' +
+          '      HW_PYRO_DUMP\n' +
```

Comment:
Should this command be HDW_BLENDER_DUMP to match the new command you added to banananation?

Reply:
The same type of thing, but we going the other way from (eDSL -> SeqJson). We are mostly testing the Class methods of `HardwareStem` and these test aren't relying on a Command Dictionary.

Reply:
👍